stock_picking_package_preparation_value/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Stock Picking Package Preparation Value
+=======================================
+
+ * Adds value to package preparation from prickings.
```

Comment:
Which value do you add? sales price value, current average Cost? Purchase value?
This is a bit ambiguous here